src/main/java/com/epam/aidial/cfg/dao/mapper/InterceptorEntityMapper.java

```diff
@@ -57,14 +62,46 @@ protected List<String> mapApplicationsAndModelsToStrings(InterceptorEntity entit
                 .collect(Collectors.toList());
     }
 
+    @Named("mapSource")
+    protected InterceptorSource mapSource(InterceptorEntity entity) {
+        InterceptorRunnerEntity runnerEntity = entity.getInterceptorRunner();
+        InterceptorContainerEntity containerEntity = entity.getInterceptorContainer();
+
+        if (runnerEntity != null && containerEntity != null) {
+            throw new IllegalStateException("Interceptor cannot have both runner and container set");
```

Comment:
add context in ex message. (e.g. entityId)

Reply:
done